main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    # create
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
There is a typo in the genre list: 'Dramma' should be 'Drama'. Although this is corrected later in the update section, it would be better to fix it here to avoid unnecessary updates.